msteams-platform/concepts/deploy-and-publish/apps-upload.md

```diff
@@ -82,18 +82,25 @@ Teams provides several ways to open apps. For more information, see [access your
 
 You don't have to sideload your app again if you make code changes (these are reflected in Teams in real-time). However, you must reinstall if you change any app configurations.
 
-If an update is available to your app, then the **Update available** option is enabled. To update, follow the steps:
+When you update an app in one thread, the new version of the app is automatically updated in all other installed contexts. Following are the steps to update an app:
 
-1. Select **Update available** to view update.
+1. Go to **Teams apps store** > Select **Manage your apps**.
 
-     :::image type="content" source="~/assets/images/publish-app/update-available.png" alt-text="Update Teams app.":::
+     :::image type="content" source="../../assets/images/publish-app/manage-apps1.png" alt-text="Screenshot is an example that shows the selection of manage your app."lightbox="../../assets/images/publish-app/manage-apps.png":::
+
+doc-feature-contextless-apps-upgrade-once-upgrade-everywhere
+    You can view updates available for apps.
 
 1. Select **View update**. A window with update option appears.
 1. Select **Update** button to update your app.
 
-     :::image type="content" source="~/assets/images/publish-app/update-window.png" alt-text="Update Teams app in manage apps.":::
+     :::image type="content" source="../../assets/images/publish-app/updates-available1.png" alt-text="Screenshot is an example that shows users can view updates available."lightbox="../../assets/images/publish-app/updates-available.png":::
+
+1. Select **Update all**, The apps are updated across all the installed contexts in Teams.
```

Comment:
The sentence does not read right. `Select Update all, The apps are updated across all the installed contexts in Teams.`
Rewrite.